openquake/calculators/tests/classical_test.py

```diff
@@ -165,6 +165,7 @@ def test_case_14(self):
 
     @attr('qa', 'hazard', 'classical')
     def test_case_15(self):  # full enumeration
+        check_platform('xenial', 'trusty')
```

Comment:
This means that we are skipping this test/these tests on RHEL too. For now it is ok, but we should think about it